src/Tokenizer/Token.php

```diff
@@ -67,6 +67,17 @@ public function __construct($token)
         }
     }
 
+    public function __debugInfo()
+    {
+        return array(
+            "\0+\0content" => $this->content,
```

Comment:
hmm weird, lets drop this PR for now, I feel like its not worth it to get my head around ;)